docs/developer-docs/defi/rosetta/icrc_rosetta/index.mdx

```diff
@@ -0,0 +1,154 @@
+When the node is caught up, you should see output similar to the following:
+
+    Synced Up to block height: 1357644
+
+After that ICRC Rosetta will update the account balances. Please wait for any queries that include fetching account balances until this process is done. 
+The corresponding message you will see is. 
+
+    Account Balances have been updated successfully
+
+:::
+
+
+
+After completing this step, the node continues to run as a **passive** node that does not participate in block making.
+
+### Step 4:  Open a new terminal window or tab and run the `ps` command to verify the status of the service.
+
+If you need to stop the service, press <kbd>Ctrl-C</kbd>. You might want to do this to change the canister identifier you are using, for example.
+
+To test the integration after setting up the node, you will need to write a program to simulate a principal submitting a transaction or looking up an account balance.
+
+### Run the Rosetta node in production
+
+When you are finished testing, you should run the Docker image in production mode without the `--interactive`, `--tty`, and `--rm` command-line options.
+These command-line options are used to attach an interactive terminal session and remove the container, and are primarily intended for testing purposes.
+To run the software in a production environment, you can start the Docker image using the `--detach` option to run the container in the background and, optionally, specify the `--volume` command for storing blocks.
+
+For more information about Docker command-line options, see the [Docker reference documentation](https://docs.docker.com/engine/reference/commandline/run/).
+
+### Requirements and limitations
+
+The integration software provided in the Docker image has one requirement that is not part of the standard Rosetta API specification.
+
+For transactions involving ICRC-1 tokens, the unsigned transaction must be created less than 24 hours before the network receives the signed transaction.
+The reason is that each transaction's `created_at` field refers to an existing transaction (essentially `last_index` available locally at the time of transaction creation). More on this topic can be read on the section on [deduplication](https://github.com/dfinity/ICRC-1/blob/main/standards/ICRC-1/README.md#transaction_deduplication).
+Any submitted transaction that refers back to a too old transaction is rejected to maintain operational efficiency.
+
+Other than this requirement, the Rosetta API integration software fully complies with all standard Rosetta endpoints and passes all of the `rosetta-cli` tests.
+The software can accept any valid Rosetta request.
+However, the integration software only prompts for transactions to be signed using Ed25519 and SECP256k1, rather than all the signature schemes [listed here](https://www.rosetta-api.org/docs/models/SignatureType.html#values), and only replies with a small subset of the potential responses that the specification supports.
```

Comment:
```suggestion
However, the integration software only prompts for transactions to be signed using Ed25519 and SECP256k1, rather than all the [signature schemes listed](https://www.rosetta-api.org/docs/models/SignatureType.html#values), and only replies with a small subset of the potential responses that the specification supports.
```